gamla/functional_generic.py

```diff
@@ -633,11 +614,11 @@ async def apply_spec_async(*args, **kwargs):
             )(spec)
 
         return apply_spec_async
-    return compose_left(
-        apply_utils.apply,
-        lambda applier: map_dict(operator.identity, applier),
-        apply_utils.apply(spec),
```

Comment:
is that function the slow function? asking because it is still used in `apply_spec_async`

Reply:
no this one is fine.

the problematic ones are the ones that check if a function is async. if this happens one time it's negligible but if it's inside a loop, it becomes subsantial

iirc here the ones that did it was the generic valmap and map